src/App.js

```diff
@@ -8,30 +8,13 @@ import htmlContent from './content.json';
 import './App.css';
 
 function App (){
-  // constructor(props){
-  //   super(props);
-  //   this.state= {
-  //     htmlHeader:'',
-  //     contentTitle:'',
-  //     contentDescription:'',
-  //     contentArray:'',
-  //     nextPageLink:''
-  //   }
-  // }
+  
   const [htmlHeader,setHtmlHeader]=useState('');
```

Comment:
Take care of the eslint requirement.
It is better to write it like this:
const [htmlHeader, setHtmlHeader] = useState('');